packages/generated-project/overrides/app/pages/cart/partials/cart-title.jsx

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React from 'react'
+import OriginalCartTitle from '@salesforce/retail-react-app/app/pages/cart/partials/cart-title'
+import AboveCartTitle from './cart-title-above'
+
+const CartTitle = () => {
+    return (
+        <>
+            <AboveCartTitle />
+            <OriginalCartTitle />
```

Comment:
When we have broken down large files into smaller ones, we can do things like this: extending an existing component like CartTitle and adding new content above or below it.

This is similar to the idea of template hooks. But the difference is we don't need to add them ourselves (and attempt to foresee the different places where template hooks may be needed). Other developers can simply add them on their own.